modules/social_features/social_event/social_event.install

```diff
@@ -1621,3 +1621,36 @@ function social_event_update_8904() {
 function social_event_update_8905() {
   \Drupal::service('module_installer')->install(['social_event_addtocal']);
 }
+
+/**
+ * Updated field_enroll_method text and labels.
+ */
+function social_event_update_8906() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_event', 'social_event_update_8906');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Update the enrollment method text of an event on event add form.
+ *
+ * Load in a config file from an specific update hook that will never change.
+ * Update helper did not update all fields correctly.
+ */
+function social_event_update_8907() {
+  $config_file = drupal_get_path('module', 'social_event') . '/config/update/social_event_update_8907.yml';
```

Comment:
These statically yml files are better of in a separate folder. eg

`/config/static/social_event_update_8907.yml`

Reply:
This so we can distinguish those from the update helper.

The downside though is now if anybody has added custom storage options, they are removed.
Maybe you are better of just writing your own update hook that loops over the labels similar to 

https://github.com/goalgorilla/open_social/pull/2096/files#diff-76261ef8588b9c0e238662e1597bf7c1dc0d0db483c3e21fd3fb897e8feb6976R1642